ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -507,14 +507,9 @@ public void delete(List<MessageInfo> infosToDelete) throws StoreException {
         List<MessageInfo> updatedInfos = new ArrayList<>(infosToDelete.size());
         int ind = 0;
```

Comment:
nit: could you change `ind` to `index`?

Reply:
we already have a member variable named "index", it's persistentIndex in the blobstore. I guess I can just name this variable to "i" if you desire.

Reply:
yeah, "i" make less ambiguity.